packages/core/src/indexing/migrations/1-create-model-table.ts

```diff
@@ -32,6 +32,53 @@ function getIndexName(tableName: string): string {
   return tableName.substring(tableName.length - 10)
 }
 
+/**
+ * Default configuration of Compose DB functionality per network.
+ * Values can be overwritten by updating them in the ceramic_config table
+ * and by restarting the node.
+ */
+function getNetworkDefaultConfig(networkName: string): { [key: string]: any } {
+  switch (networkName) {
+    case 'mainnet':
+      return {
+        is_hist_sync: true,
```

Comment:
Could we rename this flag to `enable_historical_sync` or something more explicit like this please? `support_historical_sync` maybe if it covers the functionality better?
Also for naming consistency with `allow_queries_before_historical_sync`, using `historical` rather than `hist` if possible?

Reply:
@PaulLeCam I'm good renaming it and will update the column name as part of it in the `ceramic_models` table. `enable_historical_sync` describes the intended functionality better.